xml/storage_multipath.xml

```diff
@@ -2676,97 +2678,94 @@ prio_args ""</screen>
-    <varlistentry>
-        <term>devname</term>
-        <listitem>
-            <para>
-                Regex is in device name format. For example: sda, sd.e
-            </para>
-        </listitem>
-    </varlistentry>
-    <varlistentry>
-        <term>serial</term>
-        <listitem>
-            <para>
-             Regex is in serial number format. For example: .*J1FR.*324. Look up your serial
-             number with the <command>multipathd show paths format %z</command> 
-             command.
-             (<command>multipathd show wildcards</command> displays all 
-             <literal>format</literal> wildcards.)
-            </para>
-        </listitem>
-    </varlistentry>
-
-    <varlistentry>
-        <term>alua</term>
-        <listitem>
-            <para>
-                If <literal>exclusive_pref_bit</literal> is set for a device 
-                (<literal>alua exclusive_pref_bit</literal>), paths with
-                the <literal>preferred path</literal> bit set will 
-                always be in their own path group.
-            </para>
-        </listitem>
-    </varlistentry>    
-
-<varlistentry>
-    <term>path_latency</term>
-     <listitem>
-          <para>
-               <literal>path_latency</literal> adjusts latencies
-                between remote and local storage arrays, if both arrays
-                use the same type of hardware. Usually the latency on 
-                the remote array will be higher, so you can tune the latency 
-                to bring them closer together. This requires a value pair of the form 
-                <literal>io_num=<replaceable>20</replaceable> 
-                base_num=<replaceable>10</replaceable></literal>.
-            </para>
-            <para>
-                <literal>io_num</literal> is the number of read IOs sent to the current path 
-                continuously, which are used to calculate the average path latency. Valid values 
-                are integers from  2 to 200.
-            </para>
-            <para>   
-              <literal>base_num</literal> is the logarithmic base number, 
-              used to partition different priority ranks. Valid values are integer from 2 - 10. The 
-              maximum average latency value is 100s, the minimum is 1us. For example,  
-              if  <literal>base_num=10</literal>, the paths will be grouped in priority groups with 
-               path  latency &lt;=1us, (1us, 10us], (10us, 100us], (100us, 1ms], 
-              (1ms, 10ms], (10ms, 100ms], (100ms, 1s], (1s, 10s], (10s, 100s], >100s
-             </para>
-        </listitem>
-    </varlistentry>
-</variablelist>
-  
+     These are the arguments for the prioritizer programs that require
+     arguments. Most <literal>prio</literal> programs do not need arguments.
+     There is no default. The values depend on the <literal>prio</literal>
+     setting and whether the prioritizer requires any of the following
+     arguments:
```

Comment:
I hope this block is only an outdenting change, because it's hard to compare the before/after here. (Ideally, reformatting changes should be in a different commit to make them easier to see.)